JupiterBroadcasting/jupiterbroadcasting.com
# Feat/redo json to pages/110

Closes #110 , will definitely need people to test this locally first.

Way to test locally:
- pull down my scraper repo and checkout this branch: https://github.com/elreydetoda/show-scraper/tree/feat/json_to_hugo_pages
- point data dir to be the base directory for this repo (change the first part of the volumes in the docker-compose file to be the root of this directory)
  - I'd also recommend you add this to your `.gitignore`, so you IDE doesn't cry when 100s of files get added to your disk
     ```
     content/people
     content/sponsors
     content/show
     tmp/
     ```
- Run the scrapper
- Launch Hugo
- Delete the data folders
- Re-launch hugo (to ensure no residual settings are configured)

We'll also need to coordinate with @kbondarev, so that the scaper can be merged at the same time as the main repository. This'll be important, so that way neither version of the blog is broken (the update script or just the regular blog).


Also, @gerbrent you'll be happy that I was able to unify a bunch of the views (like you mentioned here: https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/issues/145#issuecomment-1207232805) for different things. So, now there is a `partials/people` folder and all the stuff from `partials/hosts/` & `partials/guests/` were merged into that :grin:
